src/wrapper/data_chunk.ml

```diff
@@ -28,6 +28,30 @@ let get_opt t type_ idx =
 ;;
 
 let free t ~here = Resource.free t.data_chunk ~here
+let column_count count _t = count
+
+let to_string_hum ?(bars = `Unicode) col_count t =
+  (* Simplified implementation that doesn't rely on Vector.get_type *)
+  (* If there are no rows, return an empty string *)
+  if t.length = 0
```

Comment:
Let's pattern match on `t.length` and `col_count` at once, which will make the checks here a bit cleaner.

Reply:
Please address this issue.